### DISCUSSION TITLE
Dex Share Follow stopped working

### DISCUSSION THREAD

**Discussion by andersgb1** [NightscoutFoundation/xDrip#3162] [Bug reports]
First of all, massive thanks for the arguably most useful app out there!

Since last night 2023-11-01, dexcom seemingly shut down sending data to the endpoint that is being used for the Dex Share Follower source (shareous1.dexcom.com, since I'm outside US).

I checked using curl (https://gist.github.com/StephenBlackWasAlreadyTaken/adb0525344bedade1e25?permalink_comment_id=4001119#read-bg-data), and indeed, the data is not updated.

Would this work by switching to the share2-endpoint? And does it even work outside US?

**Comment by Navid200** [comment]
Hopefully, someone with experience with Dex share will respond.

Have you considered using xDrip share if master and follower both are Android, or Nightsocut if the follower is not?

**Reply by andersgb1** [reply]
Unfortunately, I cannot easily change the uploader share device, it's bound to the dexcom G6 app: https://apps.apple.com/us/app/dexcom-g6/id1209262925

**Reply by Navid200** [reply]
I'm not sure what you mean by that.  xDrip can collect directly from a G6 with no need to the Dexcom app.
And you will be able to share using xDrip sync or Nightscout.
That's what many of us have been doing for years.

**Reply by Navid200** [reply]
Or, is that your way of saying that the master is not Android.  In that case, never mind.

**Reply by andersgb1** [reply]
Exactly

**Comment by andersgb1** [comment]
Apparently a recent update to the dex receiver app caused this. It was fixed by disabling sharing, waiting 10min and then restarting the share again.

Closing this now.
